iceberg/iceberg-handler/src/test/queries/positive/query_iceberg_metadata_of_partitioned_table.q

```diff
@@ -44,10 +44,10 @@ select summary from default.ice_meta_3.snapshots;
 select summary['changed-partition-count'] from default.ice_meta_2.snapshots;
 select partition_spec_id, partition_summaries from default.ice_meta_2.manifests;
 select partition_spec_id, partition_summaries[1].upper_bound from default.ice_meta_3.manifests;
-select * from default.ice_meta_2.partitions;
-select * from default.ice_meta_3.partitions;
+select `partition` from default.ice_meta_2.partitions;
```

Comment:
http://ci.hive.apache.org/blue/organizations/jenkins/hive-precommit/detail/PR-4824/3/tests
Failed tests show that iceberg partition has three properties `last_updated_at,total_data_file_size_in_bytes,last_updated_snapshot_id` which will change over time, and we can not masked it, so I choose a fixed value `partition` in here to avoid flaky test.